docs/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Is this UserStory done?
+
+You might be confused about what it really means for a User Story to be 'done'.
+
+Well you're in luck! We at the **DoD - Department of Done** have comprised a helpful checklist to see if your code meets our one true definition of done.
+
+The User Story is done if **and only if**:
+
+- [ ] User Story
+- - [ ] All Acceptance Criteria met
+- [ ] Code
+- - [ ] No more code needed
+- - [ ] No known bugs
+- [ ] Clean Code
+- - [ ] Comments have been created for new code
+- - [ ] ESLinter found no issues
+- - [ ] No unavoidable Code Smells
+- [ ] Testing
+- - [ ] Existing Tests pass
+- - [ ] New Unit Tests created
+- - [ ] New System Tests (if possible)
```

Comment:
I would create a separate check list that is optional. And also, what are system tests?